just_audio_windows/windows/player.hpp

```diff
@@ -222,8 +222,11 @@ class AudioPlayer
         } else if (method_call.method_name().compare("setPreferredPeakBitRate") == 0) {
           result->Success(flutter::EncodableMap());
         } else if (method_call.method_name().compare("seek") == 0) {
-          const auto* position = std::get_if<int64_t>(ValueOrNull(*args, "position"));
-
+          auto* position = std::get_if<int>(ValueOrNull(*args, "position"));
+          
+          if (position == nullptr) {
+            position = std::get_if<int_64>(ValueOrNull(*args, "position"));
```

Comment:
int_64 isn't a type.

```suggestion
            position = std::get_if<int64_t>(ValueOrNull(*args, "position"));
```